ui/components/component-library/text/text.scss

```
@@ -0,0 +1,88 @@
+@use "design-system";
+@use "sass:map";
+
+$text-variants: (
+  display: ("md"),
+  heading: ( "sm", "md", "lg"),
+  body: ("xs", "sm", "md"),
+);
+
+// Variable output mixin
+// screen size, type, size
+
+@mixin textVariables($type, $size) {
+  font-family: var(--typography-s-#{$type}-#{$size}-font-family);
+  font-weight: var(--typography-s-#{$type}-#{$size}-font-weight);
+  font-size: var(--typography-s-#{$type}-#{$size}-font-size);
+  line-height: var(--typography-s-#{$type}-#{$size}-line-height);
+  letter-spacing: var(--typography-s-#{$type}-#{$size}-letter-spacing);
+
+  @include screen-lg-min {
```

I think we want this to change at `768px`(screen-md-min)
```suggestion
  @include screen-md-min {
```